data/korath/korath missions.txt

```diff
@@ -257,3 +257,588 @@ mission "Small Scale Delivery"
+				`	"What sort of delivery?"`
+					goto mission
+					to display
+						has "Korath [F01]: A Fishy Deal: down"
+						not "Korath [F01]: A Fishy Deal: mission"
+				`	"Sure, I could deliver it for you."`
+					goto yup
+					to display
+						has "Korath [F01]: A Fishy Deal: mission"
+				`	"I wish I could help, but I have urgent matters of my own."`
+					decline
+					to display
+						or
+							has "Korath [F01]: A Fishy Deal: ship"
+							has "Korath [F01]: A Fishy Deal: down"
+							has "Korath [F01]: A Fishy Deal: mission"
+				`	(Ignore her and go about your business.)`
+					decline
+					to display
+						not "Korath [F01]: A Fishy Deal: ship"
+						not "Korath [F01]: A Fishy Deal: down"
+						not "Korath [F01]: A Fishy Deal: mission"
+			label ship
+			action
+				set "Korath [F01]: A Fishy Deal: ship"
+				log "Minor People" "Jenny Jetmaster" "Latest in a family line that has operated and maintained a replica of an ancient jet aircraft and made money on airshows. She is now on the run from the Syndicate for rescuing her persecuted uncle from Tinker"
+			`	You see her expression brighten as she walks towards you. "It's an heirloom," she says proudly, gesturing widely at the ship, "which has been in the family for centuries. My distant ancestor got a military surplus cargo plane and refurbished it. They used it as a personal craft. Each generation added to it; every piece has been repaired or replaced over the last thousand years. It's more of a replica now.`
+			`	"The whole thing has been modernized - about a dozen times. There's a hyperdrive built into the hull itself, and those ion engines aren't of Syndicate design. They're custom-built, with ion propulsion, steering, and atmospheric anti-grav, in one assembly. The design was once called a Jetmaster, and my family have taken the Jetmaster name as our own.`
+			`	"Jenny Jetmaster," she says, with a choreographed bow. "My family used to do air shows all across human space, until my uncle got on the bad side of the Syndicate. We flew all the way from Bourne to Tinker to pick him up and smuggled him out to New Tortuga where the Syndicate cannot reach us. Now I smuggle cargo through hyperspace in a 20th century aircraft!"`
+			branch questions
+				has "Korath [F01]: A Fishy Deal: down"
+			label down
+			action
+				set "Korath [F01]: A Fishy Deal: down"
+			`	Her face turns sour and she says, "Unfortunately, I can't finish my current delivery. My mother is quite sick, and I want to get to Freedom before..." she trails off. "Well, a client really needs this delivery, and I have to find someone to do it."`
+				goto questions
+			label mission
+			action
+				set "Korath [F01]: A Fishy Deal: mission"
+			`	"It's the sort of delivery where you don't ask what sort of delivery it is. Take <tons> of fish to <planet>, and fly back here with <tons> of supplies. It's just two systems away, and the payment is 140,000 credits. Couldn't be easier."`
```

Comment:
I'm leaving the player in the dark on the politics so that they'll speculate. They are being given a fish delivery that is illegal, even though there's no obvious reason why it is illegal. Clearly, more is going on than they can see. Perhaps it is due to the association with these strange ships, or a jealous Syndicate, or trading endangered species, or whatever else the player may dream up.

Your "this doesn't quite make sense" feeling is what I want the player to feel.

Reply:
I rewrote the last part of the mission. How does it look now?

Reply:
I like it! Especially the little detail about the knockoff gasket, it makes it feel more real.